Tasks/UseRubyV1/installer.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+import * as task from "azure-pipelines-task-lib/task";
+import * as tool from "azure-pipelines-tool-lib/tool"
+
+enum Platform {
+    Windows,
+    MacOS,
+    Linux
+}
+
+function getPlatform(): Platform {
+    switch (process.platform) {
+        case "win32": return Platform.Windows;
+        case "darwin": return Platform.MacOS;
+        case "linux": return Platform.Linux;
+        default: throw Error(task.loc("PlatformNotRecognized"));
+    }
+}
+
+interface TaskParameters {
```

Comment:
Style nit (take it or leave it): I don't think its really necessary to have this and feels less clear than just having function parameters of version/architecture (since they're not being reused elsewhere).

Reply:
I tend to agree here. Went ahead and made the change